pkg/processor/pod/pod.go

```diff
@@ -73,14 +74,23 @@ func ProcessSpec(objName string, appMeta helmify.AppMetadata, spec corev1.PodSpe
 	}
 
 	// process nodeSelector if presented:
-	if len(spec.NodeSelector) != 0 {
+	if spec.NodeSelector == nil {
+		return specMap, values, nil
+	} else {
```

Comment:
`else` is redundant if we do return from if branch

Reply:
removed it.